contributor-guides/support-squad-guide-to-docs.md

```diff
@@ -10,7 +10,7 @@ When community members encounter Astro bugs, or have other feedback, we often do
 
 If their problem could have been helped with better documentation, please encourage them to file Docs Repo Issues too!
 
-You can point users to CONTRIBUTING.md to learn about our process, about when to file an Issue vs. a PR, what kind of PRs are helpful and likely to be accepted, as well how to fork the repository and make pull requests. Even if *you* are not contributing to the docs, you may find it helpful to review this page so that you can help facilitate contributions from others.
+You can point users to [CONTRIBUTING.md](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) to learn about our process, about when to file an Issue vs. a PR, what kind of PRs are helpful and likely to be accepted, as well how to fork the repository and make pull requests. Even if *you* are not contributing to the docs, you may find it helpful to review this page so that you can help facilitate contributions from others.
```

Comment:
I would prefer this to be the same as in `contribute.mdx`

```suggestion
You can point users to [the Main Contributing Guide](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) to learn about our process, about when to file an Issue vs. a PR, what kind of PRs are helpful and likely to be accepted, as well how to fork the repository and make pull requests. Even if *you* are not contributing to the docs, you may find it helpful to review this page so that you can help facilitate contributions from others.
```